bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java

```diff
@@ -983,4 +983,25 @@ public static String escapeHtml(@Nullable String text) {
         }
         return grouped;
     }
+    
+    public static String toASCIILetterLowerCase(String str) {
+        if (str != null && !str.trim().isEmpty()) {
+            char[] chars = str.toCharArray();
+            int i = 0;
+            for (char c : chars) {
+                if (Character.isLetter(c)) {
+                    int ic = (int) c;
```

Comment:
No need in this cast, you can use relational operators just fine with `char` type.

Reply:
Done!